internal/deb/extract.go

```diff
@@ -45,7 +45,7 @@ func checkExtractOptions(options *ExtractOptions) error {
 	return nil
 }
 
-func Extract(pkgReader io.Reader, options *ExtractOptions) (err error) {
+func Extract(fileCreator fsutil.FileCreator, pkgReader io.Reader, options *ExtractOptions) (err error) {
```

Comment:
What do you think about having the `fileCreator` inside `ExtractOptions`?

Reply:
Sure, the only thing I dont understand is why `pkgReader` is treated as a separate argument.